apps/web/src/app/settings/page.tsx

```
@@ -0,0 +1,81 @@
+        value={displayName}
+        onChange={(e) => setDisplayName(e.target.value)}
+        className="w-full rounded-md border border-gray-300 px-3 py-2 text-sm dark:border-gray-700 dark:bg-gray-900"
+        placeholder="表示名を入力（空欄でGitHub名にフォールバック）"
+      />
+      <p className="text-xs text-gray-400 mt-1">1〜50文字</p>
+
+      <button
+        onClick={handleSave}
+        disabled={saving}
+        className="mt-4 rounded-md bg-gray-900 px-4 py-2 text-sm text-white hover:bg-gray-700 disabled:opacity-50 dark:bg-white dark:text-gray-900 dark:hover:bg-gray-200"
+      >
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)